components/org.wso2.carbon.identity.application.authenticator.fido2/src/main/java/org/wso2/carbon/identity/application/authenticator/fido2/core/WebAuthnService.java

```diff
@@ -163,8 +165,14 @@
         User user = User.getUserFromUserName(CarbonContext.getThreadLocalCarbonContext().getUsername());
         user.setTenantDomain(CarbonContext.getThreadLocalCarbonContext().getTenantDomain());
 
-        PublicKeyCredentialCreationOptions credentialCreationOptions = relyingParty
-                .startRegistration(buildStartRegistrationOptions(user));
+//        PublicKeyCredentialCreationOptions credentialCreationOptions = relyingParty
+//                .startRegistration(buildStartRegistrationOptions(user));
```

Comment:
Is there a specific reason to keep this method as commented? rather than removing it?

Reply:
This was a pending task for the registration flow. Now a new API is added for this.